{"uuid":"4445958e-41f3-46ac-8980-b740dcfb4a66","historyId":"e0af38c243cd3982a430dab39535e2fc","fullName":"praktikum.courier.MakingCourierTest.courierTwoSame","labels":[{"name":"package","value":"praktikum.courier"},{"name":"testClass","value":"praktikum.courier.MakingCourierTest"},{"name":"testMethod","value":"courierTwoSame"},{"name":"suite","value":"praktikum.courier.MakingCourierTest"},{"name":"host","value":"zastya"},{"name":"thread","value":"23956@zastya.main(1)"},{"name":"framework","value":"junit4"},{"name":"language","value":"java"}],"links":[],"name":"нельзя создать двух одинаковых курьеров","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\r\nMultiple Failures (2 failures)\r\n-- failure 1 --\r\nexpected: \"Этот логин уже используется\"\r\n but was: \"Этот логин уже используется. Попробуйте другой.\"\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:51)\r\n-- failure 2 --\r\nexpected: [\"message\"]\r\n but was: [\"code\", \"message\"]\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:53)","trace":"org.assertj.core.error.AssertJMultipleFailuresError: \r\nMultiple Failures (2 failures)\r\n-- failure 1 --\r\nexpected: \"Этот логин уже используется\"\r\n but was: \"Этот логин уже используется. Попробуйте другой.\"\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:51)\r\n-- failure 2 --\r\nexpected: [\"message\"]\r\n but was: [\"code\", \"message\"]\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:53)\r\n\tat praktikum.courier.CourierChecks.checkFailedSameLogin(CourierChecks.java:54)\r\n\tat praktikum.courier.MakingCourierTest.courierTwoSame(MakingCourierTest.java:43)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)\r\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\r\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)\r\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\r\n\tat org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)\r\n\tat org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)\r\n\tat org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)\r\n\tat org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)\r\n\tat org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)\r\n\tat org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)\r\n\tat org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)\r\n\tat org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)\r\n\tat org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)\r\n\tat org.junit.runners.ParentRunner.run(ParentRunner.java:413)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)\r\n"},"stage":"finished","steps":[{"name":"создать курьера","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"courier","value":"Courier(login=zastya, password=password, firstName=user)"}],"start":1728496630278,"stop":1728496630570},{"name":"создался успешно","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.ValidatableResponseImpl@ab98032"}],"start":1728496630571,"stop":1728496630584},{"name":"залогиниться","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"creds","value":"praktikum.courier.CourierCredentials@3b48ffe8"}],"start":1728496630585,"stop":1728496631303},{"name":"залогинился","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"loginResponse","value":"io.restassured.internal.ValidatableResponseImpl@7537f638"}],"start":1728496631303,"stop":1728496631317},{"name":"создать курьера","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"courier","value":"Courier(login=zastya, password=password, firstName=user)"}],"start":1728496631317,"stop":1728496631544},{"name":"создать не получилось, такой логин уже есть","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\r\nMultiple Failures (2 failures)\r\n-- failure 1 --\r\nexpected: \"Этот логин уже используется\"\r\n but was: \"Этот логин уже используется. Попробуйте другой.\"\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:51)\r\n-- failure 2 --\r\nexpected: [\"message\"]\r\n but was: [\"code\", \"message\"]\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:53)","trace":"org.assertj.core.error.AssertJMultipleFailuresError: \r\nMultiple Failures (2 failures)\r\n-- failure 1 --\r\nexpected: \"Этот логин уже используется\"\r\n but was: \"Этот логин уже используется. Попробуйте другой.\"\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:51)\r\n-- failure 2 --\r\nexpected: [\"message\"]\r\n but was: [\"code\", \"message\"]\r\nat CourierChecks.checkFailedSameLogin(CourierChecks.java:53)\r\n\tat praktikum.courier.CourierChecks.checkFailedSameLogin(CourierChecks.java:54)\r\n\tat praktikum.courier.MakingCourierTest.courierTwoSame(MakingCourierTest.java:43)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)\r\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\r\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)\r\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\r\n\tat org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)\r\n\tat org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)\r\n\tat org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)\r\n\tat org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)\r\n\tat org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)\r\n\tat org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)\r\n\tat org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)\r\n\tat org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)\r\n\tat org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)\r\n\tat org.junit.runners.ParentRunner.run(ParentRunner.java:413)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)\r\n\tat org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.ValidatableResponseImpl@4796ae58"}],"start":1728496631545,"stop":1728496631550},{"name":"удалить курьера","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"id","value":"402053"}],"start":1728496631550,"stop":1728496631724},{"name":"удалился","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.ValidatableResponseImpl@2a85e4f4"}],"start":1728496631724,"stop":1728496631735}],"attachments":[],"parameters":[],"start":1728496630278,"stop":1728496631736}